packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -103,7 +101,7 @@ class="fixed inset-0 z-40 flex items-center min-h-screen p-4 overflow-y-auto tra
                     'max-w-5xl' => $width === '5xl',
                     'max-w-6xl' => $width === '6xl',
                     'max-w-7xl' => $width === '7xl',
-                    'fixed inset-4 w-auto' => $isFullscreen,
+                    'fixed inset-4' => $isFullscreen,
```

Comment:
I haven't checked this PR locally (and thus visually) yet, but I think a fullscreen modal should be truly fullscreen and not be inset, don't you think?

Reply:
Personally I think the little bit of space around it looks better and feels more on brand. I could probably go either way on it though.

Reply:
Hmm, I think fullscreen should be truly fullscreen. That's what you define, right?

Reply:
No worries. I'll adjust for next commit.